### PR TITLE
creating deployment overlay for solver-f35-py310

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
## Related Issues and Dependencies

Addresses: https://github.com/thoth-station/s2i-thoth/issues/231
Related to: https://github.com/thoth-station/s2i-thoth/pull/235

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Create a new solver image from recently created base image https://quay.io/repository/thoth-station/s2i-thoth-f35-py310/.

### CODE REVIEW

**Review by fridex — COMMENTED** [review]
One question, otherwise lgtm 👍🏻 thanks!

File: overlays/solver-fedora-35-py310/Pipfile.lock [hunk 1]
```diff
@@ -0,0 +1,1201 @@
+{
```
[the remark this answers is not in the document]
Comment (harshad16) [reply overlays/solver-fedora-35-py310/Pipfile.lock:1]: verified the pipfile lock in my system, seems to be the same and has the same sha.
we are good with this.

[the remark this answers is not in the document]
Comment (fridex) [reply overlays/solver-fedora-35-py310/Pipfile.lock:1]: Perfect, thanks 👍🏻

**Review by fridex — APPROVED** [review]
thanks @Gregory-Pereira 💯 👍🏻

**Review by harshad16 — CHANGES_REQUESTED** [review]
everything looks good to be merged
please include the solver in here too:

https://github.com/thoth-station/solver/blob/1d8ba2a01a276b5373fa0e4c6bae790aecbfeed9/.thoth.yaml#L6

**Comment by fridex** [comment]
I've also verified running solver testsuite in the `quay.io/thoth-station/s2i-thoth-f35-py310:v0.32.3` base. It looks like solver works well for Fedora 35 running Python 3.10.

**Review by harshad16 — APPROVED** [review]
lgtm
thanks :100:

**Pull request merged**